src/Filter/AbstractDateFilter.php

```diff
@@ -99,10 +102,19 @@ public function getRenderSettings()
             $name .= '_range';
         }
 
+        $classNames = [
```

Comment:
Can you take a look on how this is done on doctrineORM now? I think it is cleaner: https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.x/src/Filter/AbstractDateFilter.php#L158

Reply:
Done!